tests/test_backends.py

```diff
@@ -21,7 +21,11 @@
 def create_safetensors_or_bin_model_files(
     model_path: pathlib.Path, model_file_type: str, valid: bool
 ):
-    test_json_dict = {"a": 1, "b": 2}
```

Comment:
(Observation) I wonder why these tests have to write this dict into all files. Does `config.json` need any of it, except `{}`?